### ISSUE TITLE
Wrong battery capacity in Linux

### ISSUE THREAD

**Issue by plakak** [sebhildebrandt/systeminformation#569]
Hi @sebhildebrandt, it's me again - the "battery man" :joy:

**Describe the bug**
Wrong battery capacity data on a Linux machine.

**Current Output**
On left what was gathered directly from `uevent`, on right what we get in in systeminformation.
![2021-08-05-133822_1887x438_scrot](https://user-images.githubusercontent.com/13939486/128500055-f4dc4517-3698-4f4f-9f71-0e9322ce18f7.png)

After debugging, the issue is twofold - first of all the `getValue` util takes wrong value because of `line.startsWith(property)`. When checking against `POWER_SUPPLY__CHARGE_FULL_DESIGN` and `POWER_SUPPLY_CHARGE_FULL` in both cases it takes the first one. 
Second: the calculations that divide by voltage seem wrong.

In a moment I'll create a pull request with my proposal of how this could be fixed but still I'm not sure why those calculations were in the first place so I might not know the full picture. 

**Bottom line** - as I started porting my app to linux, right now I get completely wrong capacity in comparison to windows OS on the same machine, so this should be fixed :)

**Comment by plakak** [comment]
Also I see that `/sys/class/power_supply` shows a bit different results than `upower`. It's worth considering in the future to switch to `upower` for systems that have it as its values are closer to the ones I have in windows on the same machine. On the other hand `/sys/class` is a lower level and should have better info than other tools so 🤷🏻

**Comment by sebhildebrandt** [comment]
Hi Aleksander, thank you so much for making further investigation. As I am currently on vacation, I can not react as fast as I want (also concerning your pull requests). Hope this is OK.

Best regards!

**Comment by plakak** [comment]
No worries. Hope you have a great time Sebastian! I'm starting mine in a few days :)

**Comment by sebhildebrandt** [comment]
@plakak Version 5.8.1 just released. Thank you for your contribution and fix! Can you check the latest version on your side?

**Comment by plakak** [comment]
Hi @sebhildebrandt!

I'm currently on vacations, will test after 10th of September.

Thanks!

**Comment by plakak** [comment]
Hi @sebhildebrandt!

I'm currently on vacations, will test after 10th of September.

Thanks!

**Comment by plakak** [comment]
Hi @sebhildebrandt!

I'm currently on vacations, will test after 10th of September.

Thanks!

**Comment by sebhildebrandt** [comment]
@plakak take your time, enjoy your holiday! Please after returning back, have a look at my comment in your PR ...
